lib/sycamore/sycamore/transforms/mark_misc.py

```diff
@@ -116,3 +116,30 @@ def mark_break_by_tokens(parent: Document, tokenizer: Tokenizer, limit: int) ->
                 toks = 0
             toks += n
         return parent
+
+
+###############################################################################
+
+
+class MarkBBoxPreset(SingleThreadUser, NonGPUUser, Map):
+    """
+    See DocSet.mark_bbox_preset for details.
+    """
+    def __init__(self, child: Node, tokenizer: Tokenizer, token_limit: int = 512, **resource_args):
+        super().__init__(child, f=MarkBBoxPreset.mark_bbox_preset, args=[tokenizer, token_limit], **resource_args)
+
+    @staticmethod
+    @timetrace("markBBoxPreset")
+    def mark_bbox_preset(parent: Document, tokenizer: Tokenizer, token_limit: int) -> Document:
```

Comment:
The tokenizer could be large.  In the past we had problems if we tried to pass such things into an Actor in APS.  The solution in that case was to send None and default it on the callee-side.

Reply:
Done.